samples/get-all-links/get-all-links.csproj

```diff
@@ -4,7 +4,6 @@
     <OutputType>Exe</OutputType>
     <TargetFramework>netcoreapp2.1</TargetFramework>
     <RootNamespace>Example</RootNamespace>
-    <LangVersion>7.1</LangVersion>
```

Comment:
These sample projects are not part of the solution. How about setting the LangVersion to 8 in these projects?

Reply:
VS walks the dir tree for Directory.Build.props, which is at the root, so it covers all the projects

Reply:
https://docs.microsoft.com/en-us/visualstudio/msbuild/customize-your-build?view=vs-2019#search-scope